ui/tract-renderer/src/main/java/org/cru/godtools/tract/activity/TractActivity.kt

```diff
@@ -352,13 +353,15 @@ class TractActivity : BaseToolActivity<TractActivityBinding>(true, R.layout.trac
     // region Tool Pager
     private val pager get() = binding.pages
     private val pagerAdapter by lazy {
-        ManifestPagerAdapter().also {
+        ManifestPagerAdapter(pageControllerFactory).also {
             it.setCallbacks(this)
             lifecycle.addObserver(it)
             dataModel.activeManifest.observe(this, it)
         }
     }
     private var initialPage = 0
+    @Inject
+    internal lateinit var pageControllerFactory: PageController.Factory
```

Comment:
Stylistically, it seems this should be defined before its usage.

Reply:
Updated